build.yaml

```diff
@@ -169,6 +162,105 @@ common_cli:
       fi
     }
 
+    # Retries a command if it fails, using the exponential backoff algorithm.
+    # Usage:
+    # retry_n_times_with_timeout <max retries> <timeout in seconds> command
+    #
+    function retry_n_times_with_timeout {
+      set +e
+      max_tries=\${1}
+      seconds=\${2}
+      shift 2
+
+      let delay=15
+      let tries=0
+      while [ \${tries} -lt \${max_tries} ]; do
+        echo "executing...  \${*}"
+        if (timeout --foreground --signal=SIGKILL \${seconds} bash -c "\${*}"); then
+            break
```

Comment:
There is a message that shows the command failed, just a few lines below:
```
 echo Command failed, waiting \${delay}...  
```
timeout won't produce any usefull output AFAICSFTMP